lbrynet/core/Wallet.py

```diff
@@ -313,12 +314,12 @@ def get_txid_for_name(self, name):
         d.addCallback(lambda r: None if 'txid' not in r else r['txid'])
         return d
 
-    def get_stream_info_from_txid(self, name, txid):
+    def get_stream_info_from_txid(self, name, txid, nout):
```

Comment:
Changed to get_stream_info_from_claim_outpoint as outpoint is the term used in bitcoin to refer to txid + nout